tests/playwright-test/plugins.spec.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import path from 'path';
+import { test, expect } from './playwright-test-fixtures';
+
+const SIMPLE_SERVER_PATH = path.join(__dirname, 'assets', 'simple-server.js');
+
+test('should create multiple servers', async ({ runInlineTest }, { workerIndex }) => {
```

Comment:
I think this test belongs to `web-server.spec.ts`. But we should add separate tests for plugins, similar to reporters tests, that probably just log all the calls made to them and their relative order.

Reply:
moved (and added tests around interactions between the legacy config and multiple web servers w.r.t. baseURL). I'll add plugins.spec.ts that tests plugins generically in another PR.

Reply:
See https://github.com/microsoft/playwright/pull/13802.